UnityProject/Assets/Scripts/Items/Food/MeatSteak.cs

```diff
@@ -0,0 +1,25 @@
+﻿using PlayGroup;
+using UI;
+using UnityEngine.Networking;
+
+public class MeatSteak : BaseFood
+{
+    HealthBehaviour healthScript;
+    PlayerHealth playerDetails;
+    private PlayerNetworkActions playerNetworkActions;
+
+    public override void EatFood()
+    {
+
+        var currentSlot = UIManager.Hands.CurrentSlot;
+
+        currentSlot.Clear();
+
+        NetworkServer.Destroy(this.gameObject);
```

Comment:
Why didn't you use: "ClearInventorySlot(slotName);"?

Reply:
Because it doesn't work?